### PR TITLE
docs: access-control: header based access control doc

### PR DESCRIPTION
**Pull request by gioelecerati**
Still a draft because not ready to be merged until the playback endpoint support cross origin credentials

### CODE REVIEW

**Comment by gioelecerati** [comment]
This is addressing the changes related to header based access control. In the doc, we suggest to change the HLS config to set a header with the request.

@0xcadams you may want to contribute if there is a better way with the livepeer player to set headers, and how this could be done with webrtc

File: developers/guides/access-control-webhooks.mdx [hunk 1]
```diff
@@ -231,6 +231,32 @@ to the m3u8 URL as a query parameter, similar to:
...
+
+Below is an example of how to use headers when using 
+the Livepeer Player:
+
+```tsx
+import { Player } from "@livepeer/react";
+
+const hlsConfig = {
+  xhrSetup: function(xhr, url) {
+    xhr.setRequestHeader('X-Livepeer-Access-Key', 'YOUR_ACCESS_KEY');
```
Comment (victorges) [inline developers/guides/access-control-webhooks.mdx:246]: Any reason to have the `X-` prefix here but not in the other one? Maybe this one is a typo 👀

**Review by victorges — COMMENTED** [review]
LGTM but will leave for chase to approve

File: developers/guides/access-control-jwt.mdx [hunk 2]
```diff
@@ -317,3 +317,29 @@ URL as a query parameter, similar to:
...
+
+Below is an example of how to use headers when using 
+the Livepeer Player:
+
+```tsx
+import { Player } from "@livepeer/react";
+
+const hlsConfig = {
+  xhrSetup: function(xhr, url) {
+    xhr.setRequestHeader('Livepeer-Jwt', 'YOUR_JWT');
```
Comment (0xcadams) [inline developers/guides/access-control-jwt.mdx:333]: I'll update this but this shouldn't be necessary now. Let's not merge until this change is made.

**Review by 0xcadams — CHANGES_REQUESTED** [review]

File: developers/guides/access-control-webhooks.mdx (hunk 1 above)
Comment (gioelecerati) [reply developers/guides/access-control-webhooks.mdx]: Yep just a typo :D

**Pull request merged**